From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sidath Senanayake <sidaths@google.com>
Date: Tue, 1 Sep 2020 23:56:01 +0100
Subject: ANDROID: perf: Export clock_set_rate tracepoint

This allows for clock rate changes to be reported from
loadable kernel modules.

[CPNOTE: 17/06/21] Lee: Vendor related code - maintain forever

Bug: 167449286
Bug: 166776693
Change-Id: I30eb2d742aecedf3638334f85f510a69b71acb84
Signed-off-by: Sidath Senanayake <sidaths@google.com>
(cherry picked from commit daddf4fb363cbc5a183fc62fcfdd0b62748a8c26)
Signed-off-by: Will McVicker <willmcvicker@google.com>
---
 kernel/trace/power-traces.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/trace/power-traces.c b/kernel/trace/power-traces.c
--- a/kernel/trace/power-traces.c
+++ b/kernel/trace/power-traces.c
@@ -20,3 +20,4 @@ EXPORT_TRACEPOINT_SYMBOL_GPL(cpu_frequency);
 EXPORT_TRACEPOINT_SYMBOL_GPL(powernv_throttle);
 EXPORT_TRACEPOINT_SYMBOL_GPL(device_pm_callback_start);
 EXPORT_TRACEPOINT_SYMBOL_GPL(device_pm_callback_end);
+EXPORT_TRACEPOINT_SYMBOL_GPL(clock_set_rate);
